workflow-templates/test-cases-ci.yml

```diff
@@ -0,0 +1,83 @@
+name: ESP Allocation Test Cases CI  
```

Comment:
Can we change the name to generic e.g. esp-npm-test-cases-ci?

Reply:
I am changing this to ESP TEST CASES CI. Can you please confirm?

Reply:
You should prepend the BU that you work for in the name This is shared across the entire company. So maybe sws-esp-npm-test-cases-ci?